client/app/dashboard/Dashboard.js

```diff
@@ -128,6 +139,17 @@ angular.module('dashboard.Dashboard', [
     $rootScope.logOut();
     if ($event) $event.preventDefault();
   };
+
+  this.injectLocaleLanguage = function(locale) {
+    var properties = Config.serverParams.models.Account.properties;
```

Comment:
This is application specific logic. You should be able to get the language code from $translate